lib/inets/src/http_client/httpc_handler.erl

```diff
@@ -1717,6 +1718,43 @@ format_address({[$[|T], Port}) ->
 format_address(HostPort) ->
     HostPort.
 
+format_answer(Res0, Options) ->
+    FullResult = proplists:get_value(full_result, Options, true),
+    Sync = proplists:get_value(sync, Options, true),
+    do_format_answer(Res0, FullResult, Sync).
+do_format_answer({Ref, StatusLine}, _, Sync) when is_atom(StatusLine) ->
```

Comment:
make a newline between function definitions: `format_answer/2` and `do_format_answer/3`